assets/helpers/tracking/acquisitions.js

```diff
@@ -27,6 +27,7 @@ export type OphanIds = {|
 export type ReferrerAcquisitionData = {|
   campaignCode: ?string,
   referrerPageviewId: ?string,
+  otherQueryParams: [string, string][],
```

Comment:
I would not put this field here. The idea of this type is that it should match the type defined by the acquisition team.

Reply:
The definition of the type is specified in contributions-frontend [here](https://github.com/guardian/contributions-frontend/blob/1f51b86c23aee8c70248ba23016ec6ea77e651a1/app/models/ReferrerAcquisitionData.scala#L15-L26)

Reply:
Makes sense, I did wonder about this. I'll move it.